libcxx/test/std/algorithms/alg.nonmodifying/alg.fold/gaussian_sum.h

```diff
@@ -0,0 +1,19 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef LIBCXX_TEST_FOLD_GAUSSIAN_SUM_H
+#define LIBCXX_TEST_FOLD_GAUSSIAN_SUM_H
+
+#include <vector>
+
+template <class T>
+constexpr auto gaussian_sum(std::vector<T> const& input) {
```

Comment:
I'm a bit uncomfortable by the amount of assumptions this function makes about its inputs in order to produce correct output.

Reply:
Fixed.